### PR TITLE
Minor updates to the pybind wrapper

### PR DESCRIPTION
**Pull request by varunagrawal**
1. Fixed the `python-install` make command.
2. Updated the `__init__.py` file to have a common numpy import.
3. Cleaned up the `setup.py` a bit.
4.  ~~Updated the cpp template to include the stl.h header so KeyVector works~~

### CODE REVIEW

**Comment by ProfFan** [comment]
wait, I think KeyVector will not need this because it's been dealt with in stl_bind,

please refer to `python/gtsam_py/preamble.h` and `specializations.h`

**Comment by ProfFan** [comment]
Previously: `keyvector.at(i)`

Now: `keyvector[i]`

**Comment by dellaert** [comment]
@varunagrawal text @ProfFan to coordinate :-)

**Comment by varunagrawal** [comment]
The issue is wrt TBB allocators. @ProfFan said he'll add a permafix for it tomorrow.

File: python/gtsam_py/python/__init__.py [hunk 1]
```diff
@@ -1,9 +1,11 @@
 from .gtsam import *
 
+import numpy as np
```
Comment (ProfFan) [inline python/gtsam_py/python/__init__.py:3]: I think it's better to not import numpy in __init__.py so we don't get `gtsam.np`?

Comment (varunagrawal) [reply python/gtsam_py/python/__init__.py:3]: Ah that's a good point. Though I am confused why the PointX API has changed and not just handled via GTSAM?

Comment (ProfFan) [reply python/gtsam_py/python/__init__.py:3]: Point2 and Point3 are now just numpy arrays, so (

Comment (varunagrawal) [reply python/gtsam_py/python/__init__.py:3]: Huh that's strange. They were numpy arrays in the cython wrapper too, that's what eigency was for :-)

Comment (varunagrawal) [reply python/gtsam_py/python/__init__.py:3]: Let me see what I can do about it

Comment (ProfFan) [reply python/gtsam_py/python/__init__.py:3]: No they are not
```
import gtsam
p=gtsam.Point3(0,0,0)
>>> type(p)
gtsam.gtsam.Point3
>>> p.vector()
array([0., 0., 0.])
```

**Comment by dellaert** [comment]
BTW. A style comment. We should always import classes if used more than once in a unit, and nothing else.

```
import gtsam
from gtsam import Pose2

x = Pose3()
y = Pose3()
z = gtsam.Rot3()

foo = gtsam.bar(x, y, z)

```

Maybe this can go in a doc somewhere.

File: python/gtsam_py/python/__init__.py (hunk 1 above)
Comment (varunagrawal) [reply python/gtsam_py/python/__init__.py:3]: Yeah the typedefing to Vectors is the key here. I have a fix for the original numpy issue though.

Comment (ProfFan) [reply python/gtsam_py/python/__init__.py:3]: So I think this change can be reversed? After that we can merge the setup.py changes :)

**Comment by varunagrawal** [comment]
@ProfFan I added a nicer way that doesn't involve multiple import statements. This was important because potentially every time someone calls `Point2`, the numpy module is reimported, which is unnecessary overhead.

**Comment by ProfFan** [comment]
@varunagrawal Have you tested this `_init()` method? If so then good2go

**Comment by varunagrawal** [comment]
@ProfFan I did test it. I don't push till I test everything locally. :grin:

**Pull request merged**